apps/admin/backend/src/tabulation/card_counts.ts

```diff
@@ -74,8 +80,22 @@ export function tabulateScannedCardCounts({
     return cardCountsGroupMap;
   }
 
-  // general case, grouping results by specified group by clause
+  const hasExpectedGroups = groupBySupportsZeroSplits(groupBy);
+  if (hasExpectedGroups) {
+    debug('determining expected card count groups');
+    const expectedGroups = store.getTabulationGroups({
+      electionId,
+      groupBy,
+      filter,
+    });
+    for (const expectedGroup of expectedGroups) {
+      cardCountsGroupMap[getGroupKey(expectedGroup, groupBy)] =
+        getEmptyCardCounts();
+    }
+  }
+
```

Comment:
determining the expected groupings is necessary so we that expected 0 counts show up on reports.